### DISCUSSION TITLE
Development environment questions

### DISCUSSION THREAD

**Discussion by davidlag0** [dec0dOS/zero-ui#132] [General]
Hi,

I would like to contribute to this project and looking at certain issues, I would need to be able to quickly build lots of networks and connect lots of members to them. Does anyone else that contributes to this project use tools to help with this? For example, are there scripts, ansible playbooks and so on that could help me build such a virtual architecture quickly on my development machine? I would like to avoid creating myself tools first if another developer already has created such useful tools for the project that I could simply reuse. I came across issue #18 for which having such tools would help troubleshoot. I feel like I experienced the same problem with another private tool I developed with React and my solution was to use pagination and limit the number of results per page for which I ended up hardcoding quantities (10, 50, 100, etc.). Another way to fix the problem could potentially be to use lazy loading and only show items in the browser as they become part of the viewport.

Thank you!

**Comment by dec0dOS** [comment]
Hello, @davidlag0 

Currently, I have an ansible role for just installing the ZeroUI, but not managing it
https://github.com/dec0dOS/zero-ui-ansible

ZeroUI lacks an automated testing, so any help is welcome!

For additional info, please check the https://github.com/dec0dOS/zero-ui/issues/78#issuecomment-1368968044
